server/routes/custom_panels_router.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { schema } from '@osd/config-schema';
+import {
+  IRouter,
+  IOpenSearchDashboardsResponse,
+  ResponseError,
+  IScopedClusterClient,
+} from '../../../../src/core/server';
+import {CUSTOM_PANELS_API_PREFIX as API_PREFIX} from '../../common/constants/custom_panels'
+
+export function CustomPanelsRouter(router: IRouter) {
+  
+  // NOTE: Currently the API calls are dummy and are not connected to esclient.
+  // Fetch all the custom panels available
+  router.get(
+    {
+      path: `${API_PREFIX}/`,
+      validate: {},
+    },
+    async (context, request, response): Promise<IOpenSearchDashboardsResponse<any | ResponseError>> => {
+      let panelsList = [{
```

Comment:
Seems like you are not changing the value of panelsList, could you use const?

Reply:
For sure, this should be const.